### PR TITLE
new casks

### PR DESCRIPTION
**Pull request by rsanheim**

### CODE REVIEW

File: Brewfile [hunk 1]
```diff
@@ -35,11 +35,14 @@ brew "unrar"
 brew "youtube-dl"
 
 # the rest of the apps
+cask "1password"
```
Comment (rsanheim) [inline Brewfile]: 1️⃣

File: Brewfile [hunk 2]
```diff
@@ -35,9 +35,11 @@ brew "unrar"
...
 cask "audacity"
 cask "caffeine"
 cask "cloak"
```
Comment (rsanheim) [inline Brewfile]: test

Comment (dgraham) [inline Brewfile]: 🍤

**Comment by rsanheim** [comment]
> Nope

YUP!

File: Brewfile [hunk 3]
```diff
@@ -35,9 +35,11 @@ brew "unrar"
 brew "youtube-dl"
 
 # the rest of the apps
+cask "1password"
```
Comment (mattyoho) [inline Brewfile]: So useful.

**Pull request merged**